web/src/main/java/org/apache/shiro/web/util/WebUtils.java

```diff
@@ -95,6 +95,8 @@ public class WebUtils {
      */
     public static final String DEFAULT_CHARACTER_ENCODING = "ISO-8859-1";
 
+    public static final boolean IS_ALLOW_BACKSLASH = Boolean.getBoolean(ALLOW_BACKSLASH);
```

Comment:
nit: you may want to move this to just under the `ALLOW_BACKSLASH` def above in this file.

I only mention this because it took my brain a sec to realize there was an `IS_` prefix.
This isn't critical, I probably just need more caffeine 🤣